Rollbar/RollbarResult.cs

```diff
@@ -1,9 +1,44 @@
-namespace RollbarDotNet
+﻿namespace Rollbar
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    /// <summary>
+    /// Models Rollbar API response result.
```

Comment:
This object only represents successful responses, not all responses. I'm just making notes for myself so I don't forget something.

Reply:
yes. that is preserved behavior of the pre-v.1 implementation.